nilearn/decoding/space_net.py

```diff
@@ -636,12 +658,24 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
          Each pair are are the list of indices for the train and test
          samples for the corresponding fold.
 
-    `cv_scores_` : ndarray, shape (n_alphas, n_folds) or (n_l1_ratios, n_alphas, n_folds)
+    `cv_scores_` : ndarray, shape (n_folds, n_alphas) or (n_l1_ratios, n_folds, n_alphas)
         Scores (misclassification) for each alpha, and on each fold
 
     `screening_percentile_` : float
         Screening percentile corrected according to volume of mask,
         relative to the volume of standard brain.
+
+    `w_` : ndarray, shape (n_classes-1, n_features) #TODO: check shapes
+        Model weights 
+    
+    `ymean_` : array, shape (n_samples,)
+        Mean of prediction targets
+    
+    `Xmean_` : array, shape (n_features,)
+        Mean of X samples
+    
+    `Xstd_` : array, shape (n_features,)
+        Standard deviation of X samples
```

Comment:
I agree it should be mentioned that this stats are across samples.

On Thu, Dec 3, 2015 at 11:28 AM, Danilo Bzdok <notifications@github.com>
wrote:

> In nilearn/decoding/space_net.py
> <https://github.com/nilearn/nilearn/pull/881#discussion_r46534946>:
>
> >          Scores (misclassification) for each alpha, and on each fold
> >
> >      `screening_percentile_` : float
> >          Screening percentile corrected according to volume of mask,
> >          relative to the volume of standard brain.
> > +
> > +    `w_` : ndarray, shape (n_classes-1, n_features) #TODO: check shapes
> > +        Model weights
> > +
> > +    `ymean_` : array, shape (n_samples,)
> > +        Mean of prediction targets
> > +
> > +    `Xmean_` : array, shape (n_features,)
> > +        Mean of X samples
> > +
> > +    `Xstd_` : array, shape (n_features,)
> > +        Standard deviation of X samples
>
> ..."of X ACROSS samples"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/881/files#r46534946>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob